src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/calendars/ts.ui.CalendarSpirit.js

```diff
@@ -278,6 +299,11 @@ ts.ui.CalendarSpirit = (function() {
 			 */
 			_current: null,
 
+			/**
+			 * @type {Map}
+			 */
+			_noStartingValue: false,
```

Comment:
Do you need it?
If so, can you rename it to `_startingValue`?
It is hard to understand `noStartingValue = false`